modules/file-extension-common/src/main/resources/plugin.properties

```diff
@@ -1,5 +1,7 @@
 artifact.export.resources=
 
+plugin.bundle=org.mule.modules:mule-module-file-extension-common:4.0-SNAPSHOT
```

Comment:
Having the version hardcoded in this file will be a problem every time we change the version in the poms